env/fs_posix.cc

```diff
@@ -928,11 +937,19 @@ size_t PosixFileSystem::GetLogicalBlockSize(const std::string& fname, int fd) {
 #ifdef OS_LINUX
   return logical_block_size_cache_.GetLogicalBlockSize(fname, fd);
 #else
-  (void) fname;
+  (void)fname;
   return PosixHelper::GetLogicalBlockSizeOfFd(fd);
 #endif
 }
 
+size_t PosixFileSystem::GetLogicalBlockSizeIfNeeded(const EnvOptions& options,
+                                                    const std::string& fname,
+                                                    int fd) {
+  return options.use_direct_reads
```

Comment:
I believe this is needed for writes too.

Reply:
true, I realized `WritableFileWriter` will allocate aligned buffers.